README.md

```diff
@@ -86,6 +86,13 @@ For Debian/Ubuntu it would be:
 
     sudo apt install cmake pkgconf libcurl4-openssl-dev libjansson-dev libmicrohttpd-dev libsodium-dev psmisc
 
+For Red Hat/Fedora it would be:
+
+```
+sudo dnf install cmake pkgconf libcurl-devel jansson jansson-devel libmicrohttpd-devel libsodium-devel psmisc
+```
+Note: You may also need to add `git` and `gcc` to the above list if a git client and compiler, respectively, aren't already set up on your system.
```

Comment:
Any reason not to just include them in the above list?

Reply:
I imagine anyone who has a compiler preference would know to take `gcc` out, so it couldn't hurt. I was just trying to keep my command essentially the same as the one for deb.